tests/gradients/core/test_pulse_gradient.py

```diff
@@ -763,29 +763,31 @@ def circuit(params):
             for j, e in zip(jac, exp_jac):
                 assert qml.math.allclose(j[0], e, atol=tol, rtol=0.0)
 
+    @pytest.mark.xfail
     @pytest.mark.parametrize("num_split_times", [1, 2])
-    @pytest.mark.parametrize("time_interface", ["python", "numpy", "JAX"])
+    @pytest.mark.parametrize("time_interface", ["python", "numpy", "jax"])
     def test_simple_qnode_jit(self, num_split_times, time_interface):
         """Test that a simple qnode can be differentiated with stoch_pulse_grad."""
         import jax
         import jax.numpy as jnp
 
         jax.config.update("jax_enable_x64", True)
         dev = qml.device("default.qubit.jax", wires=1)
-        T = {"python": 0.2, "numpy": np.array(0.2), "JAX": jnp.array(0.2)}[time_interface]
+        T = {"python": 0.2, "numpy": np.array(0.2), "jax": jnp.array(0.2)}[time_interface]
         ham_single_q_const = qml.pulse.constant * qml.PauliY(0)
 
         @qml.qnode(
             dev, interface="jax", diff_method=stoch_pulse_grad, num_split_times=num_split_times
         )
-        def circuit(params):
+        def circuit(params, T=None):
             qml.evolve(ham_single_q_const)(params, T)
             return qml.expval(qml.PauliZ(0))
 
         params = [jnp.array(0.4)]
         p = params[0] * T
         exp_grad = -2 * jnp.sin(2 * p) * T
-        jit_grad = jax.jit(jax.grad(circuit))(params)
+        jax.jit(circuit)(params, T=T)
```

Comment:
What is with that line? To jit-compile the circuit once?

Reply:
Ah, this was from some intermediate stage where I noticed that the circuit wasn't even running, independently of differentiation. Will remove.